query-sql/src/lib.rs

```diff
@@ -323,6 +328,11 @@ impl QueryFragment for Constraint {
                 out.push_sql(")");
                 Ok(())
             },
+            &NotExists { ref subquery } => {
+                out.push_sql(" NOT EXISTS ");
```

Comment:
Doh! A remnant of a previous iteration that had the subquery outputting itself with it's braces.